.solhintignore

```diff
@@ -0,0 +1 @@
+contracts/true-currencies/tokens/TrueUSDNoDelegate.sol
```

Comment:
Ditto here: can we fix the .solhintignore reason during this upgrade? Okay to decline if this would be too large a change.

Reply:
Done.